documentation/release_notes.markdown

```diff
@@ -4,87 +4,271 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.0
```

Comment:
@nfagerlund That all sounds really complicated.  The only reason I want to change what we're doing now is to avoid merge conflicts when we merge stable up to master.  Conflicts aren't really a big deal though, especially if the release notes file is only describing the most recent release for that branch.  The conflict become pretty easy to resolve in that situation.

Seems like keeping things as they are now is going to be easier than making a change that ripples all the way through to the structure of the website itself.